R/eval-select.R

```diff
@@ -162,7 +162,21 @@ eval_select_impl <- function(x,
   local_data(x)
 
   if (length(include)) {
-    expr <- quo((all_of(include) & !(!!expr)) | !!expr)
+    # Recurse to transform `expr` into a vector of locations. This
```

Comment:
This sort of recursion seems risky because, if new arguments are added, it's easy to forget to add them here.

This encourages me to try and find a solution for https://github.com/r-lib/rlang/issues/1363.

Reply:
Before we go even further down this route, we should consider if it would be easier if `eval_select_impl` handed `include` and `exclude`.